### DISCUSSION TITLE
support `biome-expect-error`

### DISCUSSION THREAD

**Discussion by unional** [biomejs/biome#1749] [Ideas]
along with the current `biome-ignore`.

`biome-expect-error` generally works better because it can detect the issue is fixed or changed at the config,
so the extra comment can be removed promptly.

**Comment by ematipico** [comment]
That already works with `biome-ignore lint`. If the suppression isn't affecting any rule, a warning diagnostic is emitted, and you can remove the comment.

**Reply by unional** [reply]
I see. Good to know.  Thanks!
